.chloggen/feat_ecs-format.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Initial pass in implementing the `ecs` mapping mode
```

Comment:
Why is this an "initial pass"? Why not just "Implement the `ecs` mapping mode"?

Reply:
The phrasing was because this only implements a very small subset of the full ECS mapping, like this intends to expand on: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31694.